code/modules/clothing/gloves/stungloves.dm

```diff
@@ -69,6 +69,7 @@
 			user.visible_message("<span class='warning'>[user] cuts the fingertips off of the [src].</span>","<span class='warning'>You cut the fingertips off of the [src].</span>")
 
 			clipped = 1
+			siemens_coefficient += 0.25
```

Comment:
This entire proc should be moved to the generic gloves file as it is not native to the stungloves and will assuredly cause confusion again as it did today.